Clarify GPG public key extraction process and update keyserver URIs (#3024)
theforeman/foreman-documentation: guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc

[options="nowrap" subs="+quotes"]
----
$ gpg --verify Release.gpg Release
gpg: Signature made Sa 10 Feb 2024 13:41:18 CET
gpg:                using RSA key *A7236886F3CCCAAD148A27F80E98404D386FA1D9*
gpg: Can't check signature: No public key
gpg: Signature made Sa 10 Feb 2024 13:41:19 CET
gpg:                using RSA key *4CB50190207B4758A3F73A796ED0E7B82643E131*
gpg: Can't check signature: No public key
gpg: Signature made Sa 10 Feb 2024 13:46:58 CET
gpg:                using RSA key *A4285295FC7B1A81600062A9605C66F00D6C9793*

> gpg: Can't check signature: No public key

Reviewer: Yeah, maybe similar to the alternative suggestion in https://github.com/theforeman/foreman-documentation/pull/3024#discussion_r1604775148, you can add "Note the GPG key fingerprint for any missing public GPG keys." or similar.

+
[options="nowrap" subs="+quotes"]
----
$ gpg --armor --export A7236886F3CCCAAD148A27F80E98404D386FA1D9 4CB50190207B4758A3F73A796ED0E7B82643E131 A4285295FC7B1A81600062A9605C66F00D6C9793 > debian_11.txt
----
+
Ensure that `gpg` returns a `Good signature` message for each signature.
+
Upload the .txt file to {Project}.

> For more information, see xref:Importing_a_Custom_GPG_Key_{context}[import the custom GPG public key] into {Project}.

Reviewer: ```suggestion
For more information, see xref:Importing_a_Custom_GPG_Key_{context}[] into {Project}.
```

Asciidoctor will find the title of the anchor automatically.

+
[options="nowrap" subs="+quotes"]
----
$ gpg --armor --export A7236886F3CCCAAD148A27F80E98404D386FA1D9 4CB50190207B4758A3F73A796ED0E7B82643E131 A4285295FC7B1A81600062A9605C66F00D6C9793 > debian_11.txt
----
+
Ensure that `gpg` returns a `Good signature` message for each signature.
+

> Use this `.txt` file and upload it to {Project} as described below.

Reviewer 2: Maybe rephrase this to:

Upload the `.txt` file to {Project}.
For more information, see xref:Importing_a_Custom_GPG_Key_{context}[import the custom GPG public key] into {Project}.

> Use this `.txt` file and upload it to {Project} as described below.
> 
> Continue to xref:Importing_a_Custom_GPG_Key_{context}[import the custom GPG public key] into {Project}.

Reviewer 2: Maybe rephrase this to:

```suggestion
Upload the .txt file to {Project}.
For more information, see xref:Importing_a_Custom_GPG_Key_{context}[import the custom GPG public key] into {Project}.
```

Author: Sure 👍🏻
